addons/mail/static/src/models/activity_button_view.js

```diff
@@ -0,0 +1,99 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'ActivityButtonView',
+    identifyingMode: 'xor',
+    recordMethods: {
+        onClick(ev) {
+            if (this.listFieldActivityViewOwner) {
+                ev.stopPropagation(); // prevents list view click that opens form view. TODO use special_click instead?
+            }
+            if (!this.exists()) {
+                return;
+            }
```

Comment:
`this.listFieldActivityViewOwner` will crash before the purpose of `!this.exists()` below.
Do you need to `stopPropagation` before handling `!this.exists()`?

Reply:
I will move it up, even if I needed it it's impossible to code it with a view model alone.